src/OrchardCore.Modules/OrchardCore.Flows/Views/ContentCard-BagPart.Edit.cshtml

```diff
@@ -12,7 +12,7 @@
             <div class="widget-editor-handle"><i class="fas fa-arrows-alt"></i></div>
         }
         <button type="button" class="btn btn-outline-secondary btn-sm widget-editor-btn-toggle widget-editor-btn-collapse"><i class="fa fa-angle-down" aria-hidden="true"></i></button>
-        <button type="button" class="btn btn-outline-secondary btn-sm widget-editor-btn-toggle widget-editor-btn-expand"><i class="fa fa-angle-up" aria-hidden="true"></i></button>
+        <button type="button" class="btn btn-outline-secondary btn-sm widget-editor-btn-toggle widget-editor-btn-expand"><i class="fa fa-angle-right" aria-hidden="true"></i></button>
```

Comment:
It should be `fa fa-angle-left` in Arabic and all RTL languages

Reply:
We are missing a class conversion for RTL?

Reply:
I don't seem to get the current UI Culture from those views (loaded in Ajax?).
Even when I enable Localization and use `?culture=ar` in the querystring, the admin ui and menu are in RTL, but adding `@CultureInfo.CurrentCulture` in the view does not return `ar` but the default configured culture.

Reply:
What if you access the `dir` property via the DOM, because the original document should have this property, please let me know if you need a help on this

Reply:
Yes, that could be an idea.
I think I will merge this simple change and let you work on it on your own if you don't mind.

Reply:
Got for it Antoine